### ISSUE TITLE
AJAX Calls are cached when server no longer enforces Cache-Control

### ISSUE THREAD

**Issue by elmodaddyb** [gerbera/gerbera#402]
Since the removal of `Cache-Control` headers on the server, 

https://github.com/gerbera/gerbera/commit/094923d03c2b6aca20ad415410a0452a3fb6e4f1#diff-e3ba151b835166ef3bd739d2f2a36f12

all AJAX calls from the client are **cached** by default.  This causes many things to break including long-polling updates, tree view refresh, etc.

## Introduce Cache-Control on Requests
This issue is to introduce `Cache-Control` on the client-side AJAX calls similar to below
```javascript
return $.ajax({
  url: GERBERA.App.clientConfig.api,
  type: 'get',
  data: requestData,
  headers: {
    'Cache-Control': 'no-cache, must-revalidate'
  }
})
```

**Comment by elmodaddyb** [comment]
@v00d00 - I tested this locally(_still more testing required_) and it appears to resolve the caching issue.

> Since we control the client-side, it seems reasonable to include these headers.

Any thoughts?

**Comment by v00d00** [comment]
Sounds good, nicer than adding a timestamp to the request at least.

**Comment by elmodaddyb** [comment]
I found a way to set this header globally for all `$.ajax` calls...so makes it really straightforward

**Comment by v00d00** [comment]
Closed by #403
